lawnchair/src/app/lawnchair/util/PackageManagerExtensions.kt

```diff
@@ -21,3 +21,10 @@ fun PackageManager.getPackageVersionCode(packageName: String) =
     } catch (e: PackageManager.NameNotFoundException) {
         -1L
     }
+
+fun PackageManager.isPackageInstalledAndEnabled(packageName: String) =
+    try {
+        getApplicationInfo(packageName, 0).enabled
+    } catch (e: PackageManager.NameNotFoundException) {
+        false
+    }
```

Comment:
```suggestion
fun PackageManager.isPackageInstalledAndEnabled(packageName: String): Boolean =
    runCatching { getApplicationInfo(packageName, 0).enabled }.getOrDefault(false)
```

Reply:
I think we should change the style of the functions in this file in another commit outside of this PR.

Reply:
Fine.